charts/tempo-distributed/values.yaml

```diff
@@ -449,6 +447,13 @@ config: |
         service: memcached-client
         timeout: 500ms
 
+# Set Tempo server configuration
+# Refers to https://grafana.com/docs/tempo/latest/configuration/#server
+server:
+  http_listen_port: 3100
+  log_level: info
+  log_format: logfmt
```

Comment:
FYI these two are defaults already. Not sure if you intended to include them anyway

Reply:
It was intentional to explicitly let new users know that they can tweak logging through these values.

Though I can remove it if you prefer.

Reply:
OK. Might be good to add a comment that these are defaults but can be changed. 

Also CI is complaining about the docs. Once that is fixed we can merge this!